src/zone.js

```diff
@@ -429,6 +430,102 @@ Zone.prototype.createDisk = function(name, config, callback) {
   );
 };
 
+/**
+ * Create an instance group manager in this zone.
+ *
+ * @see [InstanceGroupManager Resource]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers#resource}
+ * @see [InstanceGroupManagers: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers/insert}
+ *
+ * @param {string} name - Name of the instance group manger.
+ * @param {config=} config - Configuration object.
```

Comment:
We should also add something like:

See an [InstanceGroupManager resource](https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers#resource).